docker-compose.yml

```diff
@@ -23,10 +23,19 @@ services:
       context: .
       target: development
     environment:
+      # Be sure to add environment variables to config/options.yml
       - DATABASE_URL=postgres://app:secret@postgres/app_development
       - DEV_HOST=${DEV_HOST:-localhost} # Set if you want to use a different hostname
-      - ALLOWED_REQUEST_ORIGINS=${ALLOWED_REQUEST_ORIGINS}
       - OVERMIND_COLORS=2,3,5
+      - ALLOWED_REQUEST_ORIGINS
```

Comment:
Without interpolation, these ENV variables are only set inside the container if they're set in the host. If using interpolation, then an empty string is passed in to the Docker container, which isn't ideal.

Reply:
Nice, that one is new to me.